### ISSUE TITLE
use_title_tag not work in MetadataMixin

### ISSUE THREAD

**Issue by CleitonDeLima** [nephila/django-meta#90]
Example:

```python
class MyView(MetadataMixin, TemplateView):
    use_title_tag = True
    title = 'my title'
```

META_USE_TITLE_TAG in settings has preference

**Comment by yakky** [comment]
@CleitonDeLima as all the `MetadataMixin` are defined as instance attributes, they will take precedence over class attributes defined in the derived class. Use `__init__` to set the desired values:

```
class MyView(MetadataMixin, TemplateView):
    def __init__(self, *args, **kwargs):
        super().__init__(*args, **kwargs)
        self.use_title_tag = True
        self.title = 'my title'
```

**Comment by CleitonDeLima** [comment]
@yakky Is the behavior supposed to be that way?  
Would not it be better these instance attributes are class attributes too?

**Comment by yakky** [comment]
I agree there is room for improvements, but still the capability to define them at runtime must be preserved

I don't know when I will be able to handle this, though, if anyone wants to contribute a solution, I'll happily review the PR

**Comment by CleitonDeLima** [comment]
I can help it, no problem :)

**Comment by yakky** [comment]
that would be awesome :tada:
